### PR TITLE
Capitalized smarthome concept terms

### PR DESCRIPTION
**Pull request by Confectrician**
For reference: openhab/openhab-docs#728

I have also thrown all markdown tables in the tablesgenerator.
Also some spelling errors, but only the ones i recognized while scrolling.

Signed-off-by: Jerome Luckenbach <github@luckenba.ch>

### CODE REVIEW

File: docs/documentation/concepts/categories.md [hunk 1]
```diff
@@ -6,27 +6,27 @@ layout: documentation
 
 # Categories
 
-Categories in Eclipse SmartHome are used to provide meta information about things channels, etc. UIs can use this information to render specific icons or provide a search functionality to for example filter all things for a certain category.
+Categories in Eclipse SmartHome are used to provide meta information about Things Channels, etc. UIs can use this information to render specific icons or provide a search functionality to for example filter all Things for a certain category.
```
Comment (triller-telekom) [inline docs/documentation/concepts/categories.md]: While you are at it, would you mind adding a "," behind "Things", please?

File: docs/documentation/concepts/categories.md [hunk 2]
```diff
@@ -6,27 +6,27 @@ layout: documentation
...
+The Thing type definition allows to specify a category. User interfaces can parse this category to get an idea how to render this Thing. A Binding can classify each Thing into one of the existing categories. The list of all predefined categories can be found in our categories overview:
 
 | Category        | Description | Icon Example |
 |-----------------|-------------|{% for category in site.data.categories_thing %}
 |{{category.name}}|{{category.description}}|![{{category.icon}}](../features/ui/iconset/classic/icons/{{category.icon}}){:height="36px" width="36px"}|{% endfor %}
 
 ### Channel Group Categories
 
-Channel groups can be seen as a kind of `sub-device` as they combine certain (physical) abilities of a `thing` into one. For such `group channels` one can set a category from the `thing` category list.
+Channel groups can be seen as a kind of `sub-device` as they combine certain (physical) abilities of a `Thing` into one. For such `group Channels` one can set a category from the `Thing` category list.
```
Comment (triller-telekom) [inline docs/documentation/concepts/categories.md]: maybe capitalize "group" here too

File: docs/documentation/concepts/discovery.md [hunk 3]
```diff
@@ -18,16 +18,16 @@ It is possible to override this setting and deactivate background discovery for
 
 ## Inbox
 
-The inbox holds a list of all discovered things (`DiscoveryResult`) from all active discovery services. A discovery result represents a discovered thing of a specific thing type, that could be instantiated as a thing. The result usually contains properties that identify the discovered things further like IP address or a serial number. Each discovery result also has a timestamp when it was added to or updated in the inbox and it may also contain a time to live, indicating the time after which it is be automatically removed from the inbox. 
+The inbox holds a list of all discovered Things (`DiscoveryResult`) from all active discovery services. A discovery result represents a discovered Thing of a specific Thing type, that could be instantiated as a Thing. The result usually contains properties that identify the discovered Things further like IP address or a serial number. Each discovery result also has a timestamp when it was added to or updated in the inbox and it may also contain a time to live, indicating the time after which it is be automatically removed from the inbox.
```
Comment (triller-telekom) [inline docs/documentation/concepts/discovery.md]: Could you please break up this paragraph into multiple lines, i.e. add a line break after each sentence?

File: docs/documentation/concepts/discovery.md [hunk 4]
```diff
@@ -18,16 +18,16 @@ It is possible to override this setting and deactivate background discovery for
...
+The inbox holds a list of all discovered Things (`DiscoveryResult`) from all active discovery services. A discovery result represents a discovered Thing of a specific Thing type, that could be instantiated as a Thing. The result usually contains properties that identify the discovered Things further like IP address or a serial number. Each discovery result also has a timestamp when it was added to or updated in the inbox and it may also contain a time to live, indicating the time after which it is be automatically removed from the inbox. 
 
-Discovery results can either be ignored or approved, where in the latter case a thing is created for them and they become available in the application. If an entry is ignored, it will be hidden in the inbox without creating a thing for it. 
+Discovery results can either be ignored or approved, where in the latter case a Thing is created for them and they become available in the application. If an entry is ignored, it will be hidden in the inbox without creating a Thing for it.
```
Comment (triller-telekom) [inline docs/documentation/concepts/discovery.md]: See above, please add a newline after the sentence.

File: docs/documentation/concepts/items.md [hunk 5]
```diff
@@ -45,23 +45,23 @@ Example for a Group item as a simple collection of other items:
 
 ### Derive Group State from Member Items
 
-Group items can derive their own state from their member items.
-To derive a state the group item must be constructed using a base item and a group function.
+Group Items can derive their own state from their member Items.
+To derive a state the group Item must be constructed using a base Item and a group function.
```
Comment (triller-telekom) [inline docs/documentation/concepts/items.md]: capitalize "group" too.

File: docs/documentation/concepts/profiles.md [hunk 6]
```diff
@@ -4,22 +4,22 @@ layout: documentation
 
 # Profiles
 
-The communication between the framework and the thing handlers is managed by the "Communication Manager", which in turn uses "Profiles"  to determined what exactly needs to be done. This provides some flexibility to influence these communication paths.
+The communication between the framework and the Thing handlers is managed by the "Communication Manager", which in turn uses "Profiles"  to determined what exactly needs to be done. This provides some flexibility to influence these communication paths.
```
Comment (triller-telekom) [inline docs/documentation/concepts/profiles.md]: newline after the sentence please

File: docs/documentation/concepts/profiles.md [hunk 7]
```diff
@@ -4,22 +4,22 @@ layout: documentation
...
 
-By their nature, profiles are correlated to links between items and channels (i.e. `ItemChannelLinks`). So if one channel is linked to several items it also will have several profile instances, each handling the communication to exactly one of these items. The same applies for the situation where one item is linked to multiple channels. 
+By their nature, profiles are correlated to links between Items and Channels (i.e. `ItemChannelLinks`). So if one Channel is linked to several Items it also will have several profile instances, each handling the communication to exactly one of these Items. The same applies for the situation where one Item is linked to multiple Channels.
```
Comment (triller-telekom) [inline docs/documentation/concepts/profiles.md]: newline after the sentence please

**Comment by Confectrician** [comment]
I have had a look through the whole concepts folder for the wrong newlines.
And also checked all other files for `group`. Seems that i missed that term on the first run.

**Pull request merged**